projects/angular/src/forms/datepicker/_datepicker.clarity.scss

```
@@ -74,8 +93,36 @@
     border: tokens.$cds-alias-object-border-width-100 solid datepicker-variables.$clr-calendar-border-color;
     border-radius: tokens.$cds-alias-object-border-radius-100;
     box-shadow: tokens.$cds-alias-object-shadow-100;
-    overflow: hidden;
     z-index: map.get(variables.$clr-layers, dropdown-menu);
+    justify-content: space-between;
+
+    clr-daypicker,
+    clr-monthpicker,
+    clr-yearpicker {
+      min-height: calc(100% - mixins.baselinePx(50));
+    }
+
+    &.has-action-buttons {
+      height: calc(
+        datepicker-variables.$clr-calendar-height + datepicker-variables.$clr-calendar-action-buttons-height
+      );
+    }
+
+    .action-buttons {
+      min-width: 100%;
+      height: datepicker-variables.$clr-calendar-action-buttons-height;
+      display: flex;
+      border-top: 1px solid #cbd4d8;
+      justify-content: end;
+      align-items: end;
```

In figma there is padding-top so we don't need that.
```suggestion
      padding-top: tokens.$cds-global-space-7
```